Mathlib/Data/Matrix/PEquiv.lean

```diff
@@ -50,13 +50,18 @@ variable {α : Type v}
 
 open Matrix
 
-/-- `to_matrix` returns a matrix containing ones and zeros. `f.to_matrix i j` is `1` if
+-- Porting note: added
+instance [DecidableEq n] (j : n) (o : Option n) : Decidable (j ∈ o) := by
+  rw [←Option.mem_toList]
+  infer_instance
```

Comment:
This instance shouldn't need `[DecidableEq n]`.

What instance is found in Lean3?